fluster/utils.py

```diff
@@ -161,6 +161,32 @@ def normalize_path(path: str) -> str:
     return path
 
 
+def compare_yuv_files(reference_yuv: str, test_yuv: str, tolerance: int = 2, keep_files: bool = False) -> bool:
```

Comment:
The name of this method should be changed. This is comparing byte by byte, which translates to pixel by pixel only for 8-bit YUV. This same method could be used for 8-bit RGB for example.